includes/Models/skills/wp-block-themes.md

```
@@ -1,122 +1,504 @@
+      "link":    { "color": { "text": "var(--wp--preset--color--accent)" } },
+      "button":  { "color": { "background": "var(--wp--preset--color--accent)", "text": "var(--wp--preset--color--light)" }, "border": { "radius": "0.5rem" } }
+    }
+  }
+}
+```
+
+### Typography presets — choose distinctive fonts
+
+Avoid generic fonts (Arial, Inter when used as a default). Pair a **distinctive display font** with a **refined body font**. A workable 6-step size scale:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)